lib/thredded.rb

```diff
@@ -65,11 +66,21 @@ module Thredded
   self.user_name_column = :name
   self.content_visible_while_pending_moderation = true
   self.show_topic_followers = false
+  self.messageboards_order = :position
 
   def self.user_display_name_method
     @@user_display_name_method || user_name_column
   end
 
+  def self.messageboards_order=(value)
```

Comment:
done. (all done in fact, I'll push in a second). But one point about thredded.rb @glebm -- I don't get why we're using mattr_accessors, and not cattr_accessors. Thredded is a module and is never included in class and so a mattr_accessor is a weird way of saying cattr_accessor. I think.